openmc/model/model.py

```diff
@@ -398,8 +435,31 @@ def deplete(self, timesteps, method='cecm', final_step=True,
                 depletion_operator.cleanup_when_done = True
                 depletion_operator.finalize()
 
-    def export_to_xml(self, directory='.', remove_surfs=False):
-        """Export model to XML files.
+    def export_to_xml(self, directory='.', remove_surfs=False, separate_xmls=True, path='model.xml'):
```

Comment:
@paulromano I'm wondering if we should take the same approach here as we did for `from_xml` (i.e. maintain the current `export_to_xml` method and add an `export_to_model_xml` method alongside it for now). Thoughts?

Reply:
Can do!